spec/bgs/services/manage_representative_spec.rb

```diff
@@ -43,6 +42,8 @@
   end
 
   it 'lists a single POA requests' do
+    pending('Receiving No Record Found error when hitting endpoint')
```

Comment:
This could be an incorrect assumption.
I am getting a "No Record Found" from this service, both locally and through CI.
I decided to mark it pending after noticing a similar situation here: 
https://github.com/department-of-veterans-affairs/bgs-ext/blob/64e69912c932ffb1a866a3d0fcfe06baba075db1/spec/bgs/services/claimant_spec.rb#L144

This might be a simple case of getting a new id for this to work however.